manifests/init.pp

```diff
@@ -32,6 +32,7 @@
   $action                   = 'action_mb',
   $bantime                  = 432000,
   $email                    = "fail2ban@${::domain}",
+  $sender                   = 'fail2ban@localhost',
```

Comment:
I realize this was `@localhost` before, but should this be `fail2ban@${::fqdn}` by default?

Reply:
I think that would be good.

Reply:
Fixed